CHANGELOG.rst

```diff
@@ -3,10 +3,31 @@ Changelog
 
 This document describes changes between each past release.
 
-1.3.3 (Unreleased)
+1.4.0 (2015-03-24)
 ------------------
 
-- XXX
+**Bug fixes**
+
+- Force PostgreSQl session timezone to UTC (#122)
+- Fix basic auth ofuscation and prefix (#128)
+- Make sure the `paginate_by` setting overrides the passed `limit`
+  argument (#129)
+- Fix limit comparison under Python3 (#143)
+- Do not serialize using JSON if not necessary (#131)
+- Fix crash of classic logger with unicode (#142)
+- Fix crash of CloudStorage backend when remote returns 500 (#142)
+- Fix python3.4 segmentation fault (#142)
+- Add missing port in Next-Page header (#147)
+
+
+**Internal changes**
+
+- Make monitoring dependencies optional (#121)
```

Comment:
This would be a breaking change for developpers using cliquet since last friday :)